### DISCUSSION TITLE
Execution in python interactive is extremely slow

### DISCUSSION THREAD

**Discussion by acser00** [microsoft/vscode-discussions#1365] [Extension Development QnA]
Executing python code in the interactive window is almost unusable due to its slow execution time. I am working with large dataset (11M rows) in a virtual environment on macOSx64. Any help with this issue is appreciated.

**Comment by gjsjohnmurray** [comment]
Please report this against the Python extension. The Report Issue option from the Help menu can help do this. This discussion forum focuses on extension developers.

**Comment by theReynald** [comment]
Closing due to irrelevance to this group.
